src/content/quickstart/openshiftsd/_index.md

```diff
@@ -0,0 +1,155 @@
+```
+
+
+Change the POD IP network, please note it’s a /14 range by default so you need to use 
++4 increments for “128”, for example: 10.132.0.0, 10.136.0.0, 10.140.0.0, ...
+ 
+```bash
+sed -i 's/10.128.0.0/10.132.0.0/g' cluster-b/install-config.yaml
+```
+
+Change the service IP network, this is a /16 range by default, so just use +1 increments
+for “30”: for example: 172.31.0.0, 172.32.0.0, 172.33.0.0, ...
+
+```bash
+sed -i 's/172.30.0.0/172.31.0.0/g' cluster-b/install-config.yaml
+```
+
+
+And finally deploy
+
+```bash
+openshift-install create cluster --dir cluster-b
+```
+
+### Make your clusters ready for submariner
+
+Submariner gateway nodes need to be able to accept traffic over ports 4500/UDP and 500/UDP
+when using IPSEC. In addition we use port 4800/UDP to encapsulate traffic from the worker nodes
+to the gateway nodes and ensuring that Pod IP addresses are preserved.
+
+Additionally, the default Openshift deployments don't allow assigning an elastic public IP
+to existing worker nodes, something that it's necessary at least on one end of the IPSEC connections. 
+
+To handle all those details we provide a script that will prepare your AWS OpenShift deployment
+for submariner, and will create an additional gateway node with an external IP.
+
+```bash
+
+curl https://raw.githubusercontent.com/submariner-io/submariner/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh -L -O
+chmod a+x ./prep_for_subm.sh
+
+./prep_for_subm.sh cluster-a     # respond yes when terraform asks
+./prep_for_subm.sh cluster-b      # respond yes when terraform asks
+
+```
+
+{{% notice info %}}
+
+Please note that the prep_for_subm.sh script has a few pre-requirements, you will need to install: **oc, aws-cli, terraform, and unzip**. 
+
+{{% /notice %}}
+
+### Install subctl
+
+{{< subctl-install >}}
+
+### Install Submariner with Service Discovery
+
+{{% notice info %}}
+
+The Lighthouse project is meant only to be used as a development preview. Installing the operator on an Openshift cluster may disable some of the operator features.
+
+{{% /notice %}}
+
+To install Submariner with multi-cluster service discovery follow the steps below.
+
+#### Create a merged kubeconfig
+
+```bash
+export KUBECONFIG=$PWD/cluster-a/auth/kubeconfig:$PWD/cluster-b/auth/kubeconfig
+sed -i 's/admin/east/' cluster-a/auth/kubeconfig
+sed -i 's/admin/west/' cluster-b/auth/kubeconfig
+kubectl config view --flatten > ./merged_kubeconfig
+```
+#### Use cluster-a as broker with service discovery enabled
+
+```bash
+subctl deploy-broker  --kubecontext west --kubeconfig ./merged_kubeconfig --service-discovery
+```
+
+#### Join cluster-a and cluster-b to the broker
+
+```bash
+subctl join --kubecontext west --kubeconfig ./merged_kubeconfig broker-info.subm --clusterid west --broker-cluster-context west
+```
+
+```bash
+subctl join --kubecontext east --kubeconfig ./merged_kubeconfig broker-info.subm --clusterid east --broker-cluster-context west
+```
```

Comment:
I think it'd also be nice to explain how to test it /check it works.

Reply:
Done.